README.md

```diff
@@ -7,98 +7,11 @@ log-warper
 [![Stackage Nightly](http://stackage.org/package/log-warper/badge/nightly)](http://stackage.org/nightly/package/log-warper)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
AFAIU, you changed MIT to MPL.

Reply:
By the way, why did you? I've just looked at the difference between these two and I still think MIT suits better.

Reply:
@gromakovsky @volhovm Yes, this place need to be changed. RE MIT vs. MPL-2.0. I remember some discussion in #lootbox that MPL-2.0 is better than MIT. Better contact @kirelagin for details.